### DISCUSSION TITLE
Improve PrettyBlocks UI / UX

### DISCUSSION THREAD

**Discussion by PrestaSafe** [PrestaSafe/prettyblocks#50] [Ideas]
Hi everyone, 
welcome to this first conversation. 

PrettyBlocks is fun, and we want to make it more and more funny, we think we need to improve UI and UX for a better workflow, this is why we're asking your thoughts about this improvements:

- First, make a section system like "Gutenberg" 
<img width="359" alt="Capture d’écran 2023-03-23 à 10 53 46" src="https://user-images.githubusercontent.com/4788787/227166915-73ed49b3-8162-4a74-9d1e-c6e9e0bd7ba9.png">
This would be a game changer. 

- Change or maybe remove the right side panel for a better visibilty ? 
- Add default tabs (style, advanced) and exetends this with blocks params. 
- Make some interactive component with iframe like this: 
<img width="643" alt="Capture d’écran 2023-03-23 à 10 55 29" src="https://user-images.githubusercontent.com/4788787/227167370-c2c72a1d-e07b-4d4b-a375-58a5815b0856.png">

Do you have some ideas ? Let's talk about it !

**Comment by micka-fdz** [comment]
I think we should reduce size of every interface elements, everything is too big (especially font sizes).
On your screenshot of Gutemberg, size of elements is much more consistent.

**Reply by PrestaSafe** [reply]
Ok thanks to you, For the screen shot it's only for the UX exemple: 
You put an block `title` and you can interact with it to the iframe like Gutenberg (small tool bar above it) 
:)

**Reply by micka-fdz** [reply]
I was talking about the first screenshot to compare and show that everything is too big on the current PrettyBlocks interface.

**Comment by adviva** [comment]
Hi, a grid layout solution to have by default is very important; and i'm thinking also about: 
• the possibility to select a block from the visual front
• perform some actions (for ex. delete/ a block) from a floating buttons container. Something like Elementor visual builder. 
• show a border of any block for a better visual

**Comment by MekaClaude** [comment]
_Is it possible to insert PrettyBlocks on the sidebar in the admin dashboard ?_ 

![pretty block in the sidebar backoffice](https://github.com/PrestaSafe/prettyblocks/assets/20024130/93bbdaf9-dcff-4e7c-b32a-caa8857683d9)

**Reply by PrestaSafe** [reply]
Hi, it's already done ! :) 
<img width="107" alt="Capture d’écran 2023-11-24 à 09 20 15" src="https://github.com/PrestaSafe/prettyblocks/assets/4788787/b71cfef7-614a-44c5-86e9-18d94949cafa">
